source/opt/liveness.h

```diff
@@ -69,12 +69,11 @@ class LivenessManager {
   void MarkLocsLive(uint32_t start, uint32_t count);
 
   // Return type of component of aggregate type |agg_type| at |index|
-  const analysis::Type* GetComponentType(uint32_t index,
-                                         const analysis::Type* agg_type) const;
+  uint32_t GetComponentType(uint32_t index, uint32_t agg_type_id) const;
 
   // Return offset of |index| into aggregate type |agg_type| in units of
   // input locations
-  uint32_t GetLocOffset(uint32_t index, const analysis::Type* agg_type) const;
+  uint32_t GetLocOffset(uint32_t index, uint32_t agg_type_id) const;
```

Comment:
nit: those 3 functions comments still mention "curr_type/agg_type" while function parameter changed to "curr_type_id/agg_type_id"

Reply:
https://github.com/KhronosGroup/SPIRV-Tools/pull/5699